### PR TITLE
Upgrading cordova cli and cordova android

### PR DESCRIPTION
**Pull request by wmathurin**
Also changed deployment target to 13.0 for iOS 
Also got latest sources

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -99,10 +99,10 @@
     <resource-file src="src/ios/resources/SalesforceSDKAssets.xcassets" />
 
     <!-- Mobile SDK frameworks pod specs -->
-    <pods-config ios-min-version="12.2" use-frameworks="true">
+    <pods-config ios-min-version="13.0" use-frameworks="true">
       <source url="https://cdn.cocoapods.org/"/>
     </pods-config>
-    <pod name="SalesforceHybridSDK" git="https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Hybrid" branch="dev" />
+    <pod name="SalesforceHybridSDK" git="https://github.com/wmathurin/SalesforceMobileSDK-iOS-Hybrid" branch="dev" />
```
Comment (bhariharan) [inline plugin.xml:105]: Reminder to switch this back to `forcedotcom`.

**Review by bhariharan — CHANGES_REQUESTED** [review]
Just the fork path of the repo.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Thu May 28 13:12:23 PDT 2020
+#Tue Oct 13 12:06:18 PDT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (brandonpage) [inline gradle/wrapper/gradle-wrapper.properties:6]: Is the thought here that hybrid devs probably don't care about native source and documentation?

Comment (bhariharan) [reply gradle/wrapper/gradle-wrapper.properties:6]: The default `Update` action in the latest `Android Studio` has switched from `all` to `bin` as well - I noticed this while updating another project.

Comment (brandonpage) [reply gradle/wrapper/gradle-wrapper.properties:6]: Oh, good to know.

**Pull request merged**